grammars/asciidoc.cson

```diff
@@ -48,8 +48,7 @@
   # ========================
   #
   {
-    'begin': '^\\.(\\w[\\w ]*\\w)'
-    'end': '$'
+    'match': '(?<=^\\.)(\\w.+)(?=$)'
```

Comment:
There is another PR also proposing to fix this issue. See #48.

In that PR, I describe the correct way to match a block title.

```
'begin': '(?<=^\\.)([^\\s.].*)'
```

In other words, it should match a title that is at least one character in length and does not start with a space or dot.